Index: ProjectSettings/ShaderGraphSettings.asset
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>%YAML 1.1\n%TAG !u! tag:unity3d.com,2011:\n--- !u!114 &1\nMonoBehaviour:\n  m_ObjectHideFlags: 53\n  m_CorrespondingSourceObject: {fileID: 0}\n  m_PrefabInstance: {fileID: 0}\n  m_PrefabAsset: {fileID: 0}\n  m_GameObject: {fileID: 0}\n  m_Enabled: 1\n  m_EditorHideFlags: 0\n  m_Script: {fileID: 11500000, guid: de02f9e1d18f588468e474319d09a723, type: 3}\n  m_Name: \n  m_EditorClassIdentifier: \n  shaderVariantLimit: 128\n  customInterpolatorErrorThreshold: 32\n  customInterpolatorWarningThreshold: 16\n  customHeatmapValues: {fileID: 0}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ProjectSettings/ShaderGraphSettings.asset b/ProjectSettings/ShaderGraphSettings.asset
--- a/ProjectSettings/ShaderGraphSettings.asset	(revision 013158d065ad562faa076fa6b64a8a973a5b9c69)
+++ b/ProjectSettings/ShaderGraphSettings.asset	(date 1788543418043)
@@ -13,6 +13,7 @@
   m_Name: 
   m_EditorClassIdentifier: 
   shaderVariantLimit: 128
+  overrideShaderVariantLimit: 0
   customInterpolatorErrorThreshold: 32
   customInterpolatorWarningThreshold: 16
   customHeatmapValues: {fileID: 0}
